api/notd/collection_processor.py

```diff
@@ -71,7 +71,7 @@ async def retrieve_collection(self, address: str) -> RetrievedCollection:
             contractMetadataUri = contractUriResponse[0]
         except BadRequestException:
             contractMetadataUri = None
-        if contractMetadataUri is not None:
+        if contractMetadataUri is not None and bool(contractMetadataUri):
```

Comment:
this looks super wrong i struggle to see any case in which the right hand side of this and condition gives something meaningful. looking at the contracts the cases where they dont have http:// or https:// they should be prefixed with ipfs:// (i.e. the authors have just saved the ipfs hash instead of the actual ipfs uri)

Reply:
The particular error case, the contract contract uri is an empty string i left a screenshot on slack

Reply:
oh in that case you should just do `if contractMetadataUri:` you dont need both cases

Reply:
oh okay